### ISSUE TITLE
New Question : Training Logistic Reg w/ Grad Descent

### ISSUE THREAD

**Issue by turkunov** [Open-Deep-ML/DML-OpenProblem#285]
While https://github.com/Open-Deep-ML/DML-OpenProblem/pull/283 implements a solution to a logreg-related problem, there isn't any training involved. Since I'm already on https://github.com/Open-Deep-ML/DML-OpenProblem/issues/284 , I might as well commit the problem of training a logreg as well.

* Objective of the task
  * Train Logistic Regression using gradient descent
* Input:
  * X, y
  * learning_rate
  * iterations - n of epochs
* Output:
  * list of updated parameters
  * list of recorded losses over iterations (starting from 0)

**Comment by turkunov** [comment]
Added in https://github.com/Open-Deep-ML/DML-OpenProblem/pull/286

**Comment by turkunov** [comment]
fyi @moe18
